sdk/include/psdk/ntddndis.h

```diff
@@ -440,10 +440,10 @@ typedef ULONG NDIS_OID, *PNDIS_OID;
 #define NDIS_MAC_OPTION_8021Q_VLAN                       0x00000200
 #define NDIS_MAC_OPTION_RESERVED                         0x80000000
 
-#define IOCTL_NDIS_QUERY_GLOBAL_STATS CTL_CODE(FILE_DEVICE_PHYSICAL_NETCARD, \
-                                               0,                            \
-                                               METHOD_OUT_DIRECT,            \
-                                               FILE_ANY_ACCESS)
+#define IOCTL_NDIS_QUERY_GLOBAL_STATS \
+    CTL_CODE(FILE_DEVICE_PHYSICAL_NETCARD, 0x00, METHOD_OUT_DIRECT, FILE_ANY_ACCESS) // == 0x170002
+#define IOCTL_NDIS_RESERVED7 \
+    CTL_CODE(FILE_DEVICE_PHYSICAL_NETCARD, 0x0F, METHOD_OUT_DIRECT, FILE_ANY_ACCESS) // == 0x17003e
```

Comment:
You can simplify by adding helper define, like Wine did: https://github.com/wine-mirror/wine/blob/177848adf54f6871e558be6b42ee2478a522c3d2/include/ntddndis.h#L52